core/arch/arm/kernel/boot.c

```diff
@@ -1471,8 +1515,30 @@ void __weak boot_save_args(unsigned long a0, unsigned long a1 __unused,
 	 * here. This is also called Manifest DT, related to the Manifest DT
 	 * passed in the FF-A Boot Information Blob, but with a different
 	 * compatible string.
+
+	 * Scenario B: FW Handoff via Transfer List
+	 * Note: FF-A is not yet supported with Transfer List
+	 * a0	- DTB address or 0 (AArch64)
+	 *	- must be 0 (AArch32)
+	 * a1	- TRANSFER_LIST_SIGNATURE | REG_CONVENTION_VER_MASK
```

Comment:
Regarding Armv7 platforms, this assumes that when `CFG_TRANSFER_LIST` is enabled, value 0x016ed0ff is a forbidden value for "Armv7 standard bootarg # 1" (line 1502). From mainline Linux kernel, it seems such value is not used (linux [Documentation/arch/arm/booting.rst](https://elixir.bootlin.com/linux/v6.5/source/Documentation/arch/arm/booting.rst#L63) + [arch/arm/tools/mach-types](https://elixir.bootlin.com/linux/v6.5/source/arch/arm/tools/mach-types)), but is it enough?

Regarding Armv8+ platforms, this assumes TF-A BL31 loads a1 with a value that always differs from 0x016ed0ff. I think TF-A/BL31 loads 0 in a1 but this should be cross-checked.

Reply:
Please refer to the related TF-A changes with the topics `rm/handoff` and `tl_bl31_bl32`.